iFixit/react-commerce
# Fix empty state centering bug

This PR closes #69 and fixes a bug in the Algolia library

## QA

cc @ardelato 

1. Visit the Vercel PR preview
2. Visit an empty collection page
3. Verify that the bug described by #69 is fixed
